includes/wc-product-functions.php

```diff
@@ -84,7 +84,7 @@ function wc_get_products( $args ) {
 	 * Generate WP_Query args.
 	 */
 	$wp_query_args = array(
-		'post_type'      => 'variation' === $args['type'] ? 'product_variation' : 'product',
+		'post_type'      => array( 'product', 'product_variation' ),
```

Comment:
This has reverted a change I made earlier. See lines 100 - 104 too. variations do not have 'types' - I don't even think they are linked to the taxonomy. I think this change will make the tax_query fail.

Was there a reason you changed it back?